### PR TITLE
Update Batch SDK with new comments

### PR DESCRIPTION
**Pull request by xingwu1**

### CODE REVIEW

**Comment by xingwu1** [comment]
@lmazuel Could you review it? This PR is due to minor swagger spec update (https://github.com/Azure/azure-rest-api-specs/commit/9d1643319039eee102f847d460823311c11cdc2f).

**Comment by xingwu1** [comment]
@lmazuel  ping...

File: azure-batch/azure/batch/models/node_counts.py [hunk 1]
```diff
@@ -55,6 +57,7 @@ class NodeCounts(Model):
         'running': {'required': True},
         'starting': {'required': True},
         'start_task_failed': {'required': True},
+        'leaving_pool': {'required': True},
```
Comment (lmazuel) [inline azure-batch/azure/batch/models/node_counts.py]: In order for this to be not breaking, this model has to be used only as output (i.e. from Azure to customer). Please confirm that no operation is taking this as input.

File: azure-batch/azure/batch/models/job_manager_task.py [hunk 2]
```diff
@@ -118,7 +131,7 @@ class JobManagerTask(Model):
     :type authentication_token_settings:
      ~azure.batch.models.AuthenticationTokenSettings
     :param allow_low_priority_node: Whether the Job Manager task may run on a
-     low-priority compute node. The default value is false.
+     low-priority compute node. The default value is true.
```
Comment (lmazuel) [inline azure-batch/azure/batch/models/job_manager_task.py]: Was it a doc issue?

Comment (xingwu1) [reply azure-batch/azure/batch/models/job_manager_task.py]: Yes.

File: azure-batch/azure/batch/models/node_counts.py (hunk 1 above)
Comment (xingwu1) [reply azure-batch/azure/batch/models/node_counts.py]: Yes. This is response.

**Pull request merged**